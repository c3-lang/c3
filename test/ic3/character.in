'\U+0'
'\U+1'
'\U+10'
'\U+100'
'\U+1000'
'\U+10000'
'0'
'9'
'A'
'Z'
'a'
'z'
'Π'
'ÿ'
'꒴'
'𐅀'
'🎳'
'😄'
'🟣'
'🤩'
