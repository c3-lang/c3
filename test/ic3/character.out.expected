'\0'
'\x01'
'\x10'
'Ā'
'က'
'𐀀'
'0'
'9'
'A'
'Z'
'a'
'z'
'Π'
'ÿ'
'꒴'
'𐅀'
'🎳'
'😄'
'🟣'
'🤩'
